paws.common/tests/testthat/test_handlers_query.R

```diff
@@ -708,34 +708,44 @@ test_that("unmarshal enums", {
 request <- list()
 
 test_that("unmarshal error", {
-  data <- "<ErrorResponse><Error><Code>FooError</Code><Message>Foo</Message><RequestId>123</RequestId><HostId>ABC</HostId></Error></ErrorResponse>"
+  data <- "<error_response><Error><Code>FooError</Code><Message>Foo</Message><RequestId>123</RequestId><HostId>ABC</HostId></Error></error_response>"
```

Comment:
The only thing I disagree with I think this should stay ErrorResponse.  We don't control the response, we just receive it, so I think the example response should use the actual response layout.  The name doesn't affect the result.

Reply:
Yep. I completely agree. I'll make these changes.